presidential-elections/notebooks/dynamic-model-final.ipynb

```diff
@@ -3,12 +3,16 @@
   {
```

Comment:
<p>Mmm what is this doing here if you're processing and merging the 2022 elections below?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pollsposition/models/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>No need to review the NB, it's not ready. The code to review is the python script</p>